clients/src/test/java/org/apache/kafka/clients/producer/internals/RecordAccumulatorTest.java

```diff
@@ -331,17 +331,15 @@ public void testStressfulSituation() throws Exception {
             1024 + DefaultRecordBatch.RECORD_BATCH_OVERHEAD, 10 * 1024, CompressionType.NONE, 0);
         List<Thread> threads = new ArrayList<>();
         for (int i = 0; i < numThreads; i++) {
-            threads.add(new Thread() {
-                public void run() {
-                    for (int i = 0; i < msgs; i++) {
-                        try {
-                            accum.append(topic, i % numParts, 0L, key, value, Record.EMPTY_HEADERS, null, maxBlockTimeMs, false, time.milliseconds(), cluster);
-                        } catch (Exception e) {
-                            e.printStackTrace();
-                        }
+            threads.add(new Thread(() -> {
+                for (int i1 = 0; i1 < msgs; i1++) {
```

Comment:
Nit: Does this need to be `i1` - are we iterating through the same types of things as `i` is iterating through? Can we leave it as `i` similar to the previous version or change it to `j` (or something similar)?

Reply:
Good spot! I had not realized IntelliJ had automatically named it `i1` which is a bit strange. I renamed it to `j`.